aws/components/mta/setup.ftl

```diff
@@ -109,12 +137,21 @@
                                     r'  create)',
                                     r'    info "Assign destination"',
                                     r'    aws --region "' + getRegion() + r'" ses create-configuration-set-event-destination --configuration-set-name ' + configSetName + ' --event-destination  \'${getJSON(configJSON)}\''
+                                ] + addBash + [
                                     r'    ;;'
                                     r'  update)',
                                     r'    info "Update configsets"',
                                     r'    aws --region "' + getRegion() + r'" ses update-configuration-set-event-destination --configuration-set-name ' + configSetName + ' --event-destination  \'${getJSON(configJSON)}\''
-                                    r'    ;;',
-                                    r'esac'
+                                ] + updBash + [
+                                    r'    ;;'
+                                ]+
+                                [
+                                    r'  delete)'
+                                ] + delBash + [
+                                    r'    ;;'
+                                ]+
+                                [
+                                    "esac"
```

Comment:
It might be cleaner here to just do this as a separate case statement. It looks like the addBash and updBash are the same for identities. That would also remove these generic variables which don't really say what this is for.